.github/workflows/build-bootc.yaml

```diff
@@ -31,30 +37,53 @@ jobs:
           popd
 
       - name: Set up QEMU
-        uses: docker/setup-qemu-action@v1
+        uses: docker/setup-qemu-action@v3
 
       - name: Set up Docker Buildx
-        uses: docker/setup-buildx-action@v1
+        uses: docker/setup-buildx-action@v3
+        with:
+          driver-opts: network=host
 
-      - name: Login to Quay.io
+      - name: Login to registry
         uses: docker/login-action@v3
         with:
-          registry: quay.io
+          registry: ${{ env.REGISTRY }}
           username: ${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_USERNAME }}
           password: ${{ secrets.QUAY_FLIGHTCTL_INFRA_ROBOT_PASSWORD }}
 
-      - name: Build and push
-        uses: docker/build-push-action@v2
+      - name: Build CentOS image
+        uses: docker/build-push-action@v5
         with:
           context: bootc-agent-images/centos
           file: bootc-agent-images/centos/Containerfile
           platforms: linux/amd64,linux/arm64
           push: true
-          tags: quay.io/flightctl/flightctl-agent-centos:bootstrap
+          tags: localhost:5000/user/flightctl-agent-centos:bootstrap
 
+      - name: Test CentOS image
+        uses: addnab/docker-run-action@v3
+        with:
+          image: localhost:5000/user/flightctl-agent-centos:bootstrap
+          run: |
+            rpm -qa | grep flightctl-agent
```

Comment:
This just checks that the flightctl-agent is installed I wonder if we can like start the agent temporarily for an even better check 🤔 Although this is fine for now